### PR TITLE
Adding videos to walk users through adding the app to their home screen

### LINKED ISSUE #10: Add videos to walk users through adding the app to their home screen
AC:

1. In the `Footer` should appear some help text prompting the user to add the app to their home screen
2. Below the help text should be 2 links, 1 for “How to: iOS” and 1 for “How to: Android”
3. When the user taps one of the links, it should open a modal
4. In the modal, a video should display that shows a user adding the app to their home screen for the operating system selected (you can use `public/img/screen-cast-android.mp4` and `public/img/screen-cast-ios.mp4`)

### PR DESCRIPTION
**Pull request by AshlingH**
## Description

We placed a button in the footer for the user to click, this opens up a modal where both videos are held. We applied Materialize in the modal to display the videos. This code needs further CSS changes and perhaps further structure but have a look!

## Related Issue

closes #10 

## Acceptance Criteria

- [x] In the Footer should appear some help text prompting the user to add the app to their home screen

- [x] Below the help text should be 2 links, 1 for “How to: iOS” and 1 for “How to: Android”

- [x] When the user taps one of the links, it should open a modal

- [x] In the modal, a video should display that shows a user adding the app to their home screen for the operating system selected (you can use public/img/screen-cast-android.mp4 and public/img/screen-cast-ios.mp4)

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

Screenshot -modal
Screenshot-button click

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,6 +24,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+     <!-- Compiled and minified CSS -->
+     <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"> -->
```
Comment (mjhcodes) [inline public/index.html]: This could probably be removed, if we're not using Materialize's CSS.

File: src/components/Footer.js [hunk 2]
```diff
@@ -56,6 +66,7 @@ Footer.propTypes = {
   displayInformation: PropTypes.bool.isRequired,
   handleLocationSharedClick: PropTypes.func.isRequired,
   marker: PropTypes.bool.isRequired,
+  setVideoModal: PropTypes.func.isRequired
```
Comment (mjhcodes) [inline src/components/Footer.js]: This wouldn't be required, as the `setVideoModal` function is being created within the Footer.

File: src/components/VideosModal.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {Collapsible, CollapsibleItem, Icon } from 'react-materialize';
```
Comment (mjhcodes) [inline src/components/VideosModal.js]: `Icon` can be removed from the imports, as it's not being used. That'll help clear up the console warning.

File: src/components/VideosModal.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    <div id="modal" className="modal-container">
+      {/* React Materalize import */}
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"></link>
+      <section className="modal-content">
+        <img
+          src={"/img/close-x.png"}
+          alt="close icon"
+          onClick={() => setVideoModal(false)}
+          className="modal-close-button"
+        />
```
Comment (mjhcodes) [inline src/components/VideosModal.js]: It'd be good to wrap this with a button for accessibility purposes.

File: src/components/VideosModal.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {Collapsible, CollapsibleItem, Icon } from 'react-materialize';
+
+export default function VideosModal({ setVideoModal }) {
+  return (
+    <div id="modal" className="modal-container">
+      {/* React Materalize import */}
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"></link>
```
Comment (mjhcodes) [inline src/components/VideosModal.js:9]: I had no idea we could link Materialize into just one file. I thought it was all or nothing. 😆  Clever!

File: src/components/VideosModal.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+        <h1 className="modal-header">Videos</h1>
+        <Collapsible accordion={true}>
+          <CollapsibleItem
+              expanded={false}
+              header="iOS Instructions"
+              node="div"
+          >
+            <video
+              className="App-info-how-to"
+              //poster={`/img/screen-grab-ios.png`}
```
Comment (mjhcodes) [inline src/components/VideosModal.js]: What does the `poster` prop do? Are they required?

**Review by mjhcodes — APPROVED** [review]
Great work! I love the use of the modal. It keeps it in line with the use of the Settings modal and implementing Materialize CSS for the accordion effect was clever and efficient.

During use, the videos are spilling out of the modal when expanded, so that'd be good to address from a CSS standpoint. Perhaps adding a scrolling ability or some changes to the height properties could be helpful.

File: src/components/VideosModal.js (hunk 5 above)
Comment (AshlingH) [reply src/components/VideosModal.js:9]: neither did we untill we tried it haha!

**Comment by ajiles91** [comment]
Everything looks good! Matthew was pretty through so I don't really have anything to add

**Review by ajiles91 — COMMENTED** [review]
Everything looks good. Matthew was pretty through so I think its ready for Andrew :)

File: src/components/VideosModal.js [hunk 7]
```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {Collapsible, CollapsibleItem} from 'react-materialize';
+
+export default function VideosModal({ setVideoModal }) {
+  return (
+    <div id="modal" className="modal-container">
+      {/* React Materalize import */}
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"></link>
```
Comment (segdeha) [inline src/components/VideosModal.js:9]: Huh. I [looked it up](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link) and it's actually valid to use a `<link rel="stylesheet">` within the `<body>` of an HTML file. Who knew?!

Good to know that it's not a well-accepted practice and that most (all?) other types of `<link>` tags are not allowed within the `<body>` tag.

**Review by segdeha — APPROVED** [review]
Conditionally approving to keep y'all unblocked. Let's add some CSS to get these videos sized correctly, then this PR is ready to merge! 👏 👏 👏

**Pull request merged**